### ISSUE TITLE
[diag] 7.x clusters are incorrectly reporting `minimum_master_nodes` setting in the Analyzer output

### ISSUE THREAD

**Issue by aliciascott** [elastic/support-diagnostics#305]
7.x clusters are incorrectly reporting `minimum_master_nodes` setting in the Analyzer output:

```
Minimum masters set to: -1 at node: es-master-01 / xxxxxxxx
No minimum master set
4 master nodes do not have minimium master setting - check the settings view to indentify them.```

Can we remove the check for this at `7.x` in the Analyzer to avoid confusion?
